src/constants/chains.test.ts

```diff
@@ -2,14 +2,15 @@ import { SupportedChainId as SdkSupportedChainId } from '@uniswap/sdk-core'
 
 import { ALL_SUPPORTED_CHAIN_IDS, SupportedChainId } from './chains'
 
+// These tests will be deprecated as part of the moving of ChainId to sdk-core
```

Comment:
make this a todo?

Reply:
Already handled in open PR https://github.com/Uniswap/interface/pull/6757